api/app/src/shared/config.ts

```diff
@@ -97,4 +97,20 @@ export class Config {
   static getSystemRootOID(): string {
     return getEnvVarOrFail("SYSTEM_ROOT_OID");
   }
+
+  static getGatewayEndpoint(): string {
+    return getEnvVarOrFail("CW_GATEWAY_ENDPOINT");
+  }
+
+  static getGatewayAuthorizationServerEndpoint(): string {
+    return getEnvVarOrFail("CW_GATEWAY_AUTHORIZATION_SERVER_ENDPOINT");
+  }
+
+  static getGatewayAuthorizationClientId(): string {
+    return getEnvVarOrFail("CW_GATEWAY_AUTHORIZATION_CLIENT_ID");
+  }
+
+  static getGatewayAuthorizationClientSecret(): string {
+    return getEnvVarOrFail("CW_GATEWAY_AUTHORIZATION_CLIENT_SECRET");
+  }
```

Comment:
I don't mean to be picky, but these variable names lead one to think they are related to CW's infrastructure, like if "CW_GATEWAY_ENDPOINT" was an endpoint on CW's server.

Whereas they are Metriport's infrastructure, that is used to setup Orgs on CW - but in theory could be used for other partners as well.

I see there's a bunch of other data with the `CW_` prefix that are related to Metriport, so its following that pattern. Maybe we just add a note on the README to make it clear for people who wonder what that is used for?